package.json

```diff
@@ -2,21 +2,27 @@
   "name": "node-proxy",
   "version": "1.4.6",
   "description": "Proxy",
+  "author": "pellepelle3",
+  "license": "MIT",
```

Comment:
I prefer MIT, but if people strongly suggest we use something else, I’ll be fine with that.

BTW, we literally cannot use a copyleft license since we use this in private projects.